source/backtraceMetric.ts

```diff
@@ -0,0 +1,226 @@
+import { BacktraceClientOptions } from '.';
+import { uuid } from './utils';
+
+declare const __VERSION__: string;
+
+/**
+ * Function to create instance of BacktraceMetric and ping.
+ */
+export default function backtraceMetric(
+  configuration: BacktraceClientOptions,
+): void {
+  new BacktraceMetric(configuration).ping();
+}
+
+/**
+ * Handles Backtrace Metrics.
+ * TODO: Consider converting to a static class since an instance is only created to call a single function once.
+ */
+export class BacktraceMetric {
+  public universe;
+  public token;
+  public timeout = 15000; // fifteen seconds in miliseconds
+
+  /** Seconds since epoch. */
+  public readonly CURRENT_TIMESTAMP: number = Math.floor(
```

Comment:
Shouldn't we move it to the helper? We already have a code responsible for generating a timestamp: 
https://github.com/backtrace-labs/backtrace-js/blob/master/source/model/backtraceReport.ts#L15